tests/testthat/test-ci.R

```diff
@@ -26,8 +24,21 @@ test_that("GitHub Actions functionality works in a subdirectory", {
   )
 })
 
-test_that("GitHub Actions - linting on error works", {
-  # imitate being on GHA whether or not we are
+patrick::with_parameters_test_that(
+  "GitHub Actions - env var for error on lint is converted to logical",
+  {
+    withr::local_envvar(list(GITHUB_ACTIONS = "true", LINTR_ERROR_ON_LINT = env_var_value))
+    withr::local_options(lintr.rstudio_source_markers = FALSE)
+    tmp <- withr::local_tempfile(lines = "x <- 1:nrow(y)")
+
+    l <- lint(tmp)
+
+    expect_output(print(l), "::warning file", fixed = TRUE)
+  },
+  env_var_value = c("", "F", NA),
```

Comment:
We were also testing `NULL`; I am not sure how to incorporate that in this parametrized test suite.

Reply:
We are no longer testing `"true"` in this branch, are we?

Reply:
Is NULL the equivalent of Sys.unsetenv()?
If so, there is an observable difference in behavior to `""`, so I wouldn't remove the test case.

Reply:
Sounds good. Added a test case for `NULL` as well.